superset/tags/models.py

```diff
@@ -131,7 +131,9 @@ def __str__(self) -> str:
         return f"<TaggedObject: {self.object_type}:{self.object_id} TAG:{self.tag_id}>"
 
 
-def get_tag(name: str, session: orm.Session, type_: TagType) -> Tag:
```

Comment:
I left tags as is. The way we use SQLAlchemy event handlers isn't great as i) these aren't invoked for batch operations, and ii) we're mutating records other than the record associated with that callback which requires the use of a new session. Ideally these operations should be handled via commands.